File: src/main/java/com/prmncr/normativecontrol/NormativeControlApplication.java
package com.prmncr.normativecontrol;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class NormativeControlApplication {
    public static void main(String[] args) {
        SpringApplication.run(NormativeControlApplication.class, args);
    }
}

File: src/main/kotlin/com/prmncr/normativecontrol/NormativeControlApplication.kt
package com.prmncr.normativecontrol

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync

@SpringBootApplication
@EnableAsync
class NormativeControlApplication

fun main(args: Array<String>) {
    runApplication<NormativeControlApplication>(*args)
}